api_tests/nodes/views/test_node_detail.py

```diff
@@ -821,6 +824,38 @@ def test_can_make_project_public_if_admin_contributor(
             project_private.reload()
             assert project_private.is_public
 
+    def test_make_project_private(
```

Comment:
nit: if we make this two tests, we can see which case is failing right away -- or if both cases are failing, instead of letting a failure in the "over storage" case mask a more general failure.